pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dto/mercury/MercuryOrderDto.java

```diff
@@ -0,0 +1,52 @@
+package org.broadinstitute.dsm.db.dto.mercury;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+
+import lombok.Data;
+import org.broadinstitute.dsm.db.dao.mercury.MercuryOrderDao;
+
+@Data
+
+public class MercuryOrderDto {
+    long mercurySequencingId;
+    String ddpParticipantId;
+    String orderId;
+    String creatorId;
+    String barcode;
+    int kitTypeId;
+    long orderDate;
+    String orderStatus;
+    long statusDate;
+    String mercuryPdoId;
+
+    public MercuryOrderDto(String ddpParticipantId, String creatorId, String barcode, int kitTypeId) {
+        this.ddpParticipantId = ddpParticipantId;
+        this.creatorId = creatorId;
+        this.barcode = barcode;
+        this.kitTypeId = kitTypeId;
+    }
+
+    public static List<MercuryOrderDto> createAllOrders(String[] barcodes, String ddpParticipantId, String orderId)
```

Comment:
In general `DTO` classes are classes which only hold the data and nothing more, usually there are so called `use case` classes which are responsible to do business logic, this method here is out of scope of `DTO`, I would have  class `MercuryOrderUseCase` or something like that, which will be responsible for business logic.